spack-ext/repos/spack-stack/packages/py-xnrl/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
```

Comment:
I think this is fine to put the package.py for py-xnrl here, but for my edification what is the criteria to chose this directory versus adding the package.py to the authoritative spack repo? For example are these for special packages that would not necessarily be of interest to the broader spack community?

Reply:
Correct. While there is one old release that is available to the public, none of the newer releases is. It's also a question of who would support the package to the community if it was available to everyone.